app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy > 0:
```

Comment:
according checklist you can use here just ```if masks_to_buy``` without ```> 0```. But in generally good job

Reply:
![image](https://user-images.githubusercontent.com/103201010/219680444-1b43f0f9-292b-4e63-b8c7-b02e445c09cb.png)
@usrofgh is right, fix this please